src/main/java/bio/terra/service/filedata/azure/util/BlobContainerClientFactory.java

```diff
@@ -0,0 +1,201 @@
+package bio.terra.service.filedata.azure.util;
+
+import com.azure.core.credential.AzureSasCredential;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.storage.blob.*;
```

Comment:
We don't use wildcard imports in our code, if you pick up the latest editorconfig changes from `develop` and have editorconfig support enabled it should prevent this from happening.

Reply:
Fixed.